scheduler/core/programprovider/ocs/ocsprogramprovider.py

```diff
@@ -580,11 +587,11 @@ def find_filter(filter_input: str, filter_dict: Mapping[str, float]) -> Optional
             wavelength = None
 
         # Identify GRACES
-        # if instrument == 'GRACES':
-        #     if 'DS920' in filt:
-        #         fpu = '1-fiber'
-        #     elif 'HeIIC' in filt:
-        #         fpu = '2-fiber'
+        if instrument == 'GRACES':
```

Comment:
Repeated strings like this littered throughout the code are magic values: we should probably just define a str at the class level, `graces = 'GRACES'`, and then use that in these computations instead of repeatedly referencing `'GRACES'` directly, but I'll make that change if we have time.